engine/classes/Elgg/Amd/Config.php

```diff
@@ -218,11 +218,18 @@ public function hasModule($name) {
 	 * @return array
 	 */
 	public function getConfig() {
-		return array(
+		$defaults = array(
 			'baseUrl' => $this->baseUrl,
 			'paths' => $this->paths,
 			'shim' => $this->shim,
 			'deps' => $this->getDependencies(),
+			'waitSeconds' => 20,
 		);
+		
+		$params = array(
+			'defaults' => $defaults
+		);
+		
+		return elgg_trigger_plugin_hook('config', 'amd', $params, $defaults);
```

Comment:
Big real difference would be that a test of this hook would have to test the js/elgg/require_config view, not just the Config component.